examples/scim-patch/main.go

```diff
@@ -0,0 +1,127 @@
+package main
+
+// A simple example of fetching a user by ID and using the Patch API to update
+// their name.
+//
+// Usage:
+//   go run main.go <host> <token> <user-id>
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	"github.com/databricks/databricks-sdk-go"
+	"github.com/databricks/databricks-sdk-go/logger"
+	"github.com/databricks/databricks-sdk-go/service/iam"
+)
+
+func fetchAndPrintUserName(c *databricks.AccountClient, id string) error {
+	user, err := c.Users.GetById(context.Background(), id)
```

Comment:
Ctx must always be the first parameters and initialized as much at the top of business logic as possible. Here - in main()